Presentation.Web/Controllers/API/V2/Internal/Excel/ExcelInternalV2Controller.cs

```diff
@@ -0,0 +1,230 @@
+﻿using System;
+using Core.ApplicationServices;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http.Headers;
+using System.Net.Http;
+using System.Net;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Http;
+using System.Web.Http.Results;
+using Presentation.Web.Models.API.V2.Internal.Response.Excel;
+using Swashbuckle.Swagger.Annotations;
+using Core.ApplicationServices.Model.Excel;
+using Core.Abstractions.Types;
+using Presentation.Web.Helpers;
+
+namespace Presentation.Web.Controllers.API.V2.Internal.Excel
+{
+    [RoutePrefix("api/v2/internal/organizations/{organizationUuid}/local-admin/excel")]
+    public class ExcelInternalV2Controller : InternalApiV2Controller
+    {
+        private readonly IExcelService _excelService;
+        private readonly string _mapPath = HttpContext.Current.Server.MapPath(Constants.Excel.ExcelFilePath);
+
+        public ExcelInternalV2Controller(IExcelService excelService)
+        {
+            _excelService = excelService;
+        }
+
+        #region Excel Users
+
+        [HttpGet]
+        [Route("users")]
+        [SwaggerResponse(HttpStatusCode.OK)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public IHttpActionResult GetUsers(Guid organizationUuid, bool? exportUsers)
+        {
+            return GetExcelFile(organizationUuid, Constants.Excel.UserFileName, _excelService.ExportUsers)
+                .Match(result => GetResponseMessage(result.MemoryStream, result.FileName), FromOperationError);
+        }
+
+        [HttpPost]
+        [Route("users")]
+        [SwaggerResponse(HttpStatusCode.OK)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        [SwaggerResponse(HttpStatusCode.Conflict)]
+        public async Task<IHttpActionResult> PostUsers(Guid organizationUuid, bool? importUsers)
+        {
+            var result = _excelService.ResolveOrganizationIdAndValidateAccess(organizationUuid);
```

Comment:
Lines 53-63 are the same for all three endpoints, can be extracted to a helper method.